src/crypto/poly.rs

```diff
@@ -327,6 +340,20 @@ pub struct BivarPoly {
     coeff: Vec<Fr>,
 }
 
+impl Drop for BivarPoly {
+    fn drop(&mut self) {
+        let degree_ptr = self.degree as *mut usize;
+        let coeff_ptr = self.coeff.as_mut_ptr();
+        unsafe {
+            write_volatile(degree_ptr, 0);
```

Comment:
That's unnecessary; the degree is in the commitment, too, so it can't be a secret anyway.